### ISSUE TITLE
[Deque Analysis] `<html>` element must have a `lang` attribute

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#1188]
> Serious: This issue results in serious barriers for people with disabilities, and will partially prevent them from accessing fundamental features or content. People relying on assistive technologies will experience significant frustration as a result. Issues falling under this category are major problems, and remediation should be a priority. 

## Deque Analysis Summary

Deque noted this as a site-wide issue (across the entire UI), see [ID 471067](https://axeauditor.dequecloud.com/test-run/0856438a-a19a-11eb-bc31-b7d5be387c86/issue/4f2cbf62-a3fe-11eb-883c-671d38276f41?sortField=ordinal&sortDir=asc&filter%5Bseverity%5D=3&filter%5Btype%5D=issue&filter%5Bpage_number%5D=16&row=0) (requires login)

Simply put, the `<html>` tag (at the root of every page) should have a `lang` attribute.  That attribute probably should match the selected language (based on the language/globe dropdown in the header). 

More information: https://dequeuniversity.com/rules/axe/4.1/html-has-lang?application=axeAPI

**Comment by artlowel** [comment]
This wil take an estimated **1 hour**

**Comment by tdonohue** [comment]
Thanks @artlowel , labelled as such & moved to in progress for your team to work on.
